src/ui/pages/Menu/Menu.types.ts

```diff
@@ -5,6 +5,10 @@ interface SubMenuData {
   title: string;
   additionalButtons: ReactNode;
   pageId: string;
+  props?: {
+    switchView: (key: SubMenuKey) => void;
+  };
```

Comment:
I don't get this type. Shouldn't it be this?
```
Component: (props?: { switchView: (key: SubMenuKey) => void }) => ReactElement;
```

Reply:
That works as well, changing it now 👍